### DISCUSSION TITLE
关于 ShokaX 寻找社区维护者的通知

### DISCUSSION THREAD

**Discussion by zkz098** [theme-shoka-x/hexo-theme-shokaX#137] [Announcements]
我(即zkz098)由于现实生活原因，预计在`2023.9-2024.7`这个时间段内将会很少参加开发和社区管理。因此，为避免 ShokaX 出现长时间无维护问题和分摊社区维护者压力，ShokaX 开始寻找社区维护者。
只要你满足以下条件，就可以申请维护者:
- 为 ShokaX 提供过 commit 且 包含在正式版中
- 为 ShokaX Docs 或 子项目提供过commit
- 为 ShokaX 提供过 issue 或建议且最后包含在正式版中(或在正式版中修复)

第三条需要与第一或第二条结合使用
只需要在本讨论下回复即可(可以at一下我)，我会在确认后添加member权限

**Comment by foryouos** [comment]
- 作为shokaX的忠实使用者，也特别喜欢这个主题，也非常感谢开发者在百忙之中无偿发电，我也特别希望能够加入到社区维护建设中，并希望能为ShokaX越来越完善献出微薄之力。
- 满足第二个，第三个条件
@zkz098

**Reply by zkz098** [reply]
已发送invite，通过即可加入

**Reply by foryouos** [reply]
好的，已同意，加进来了。您好，请问方便加个联系方式吗？   foryouos@qq.com   wx /  QQ

**Reply by zkz098** [reply]
已发邮件

**Comment by zkz098** [comment]
今天开始我就没法处理 Issue 和 代码更改了，可能会参与回复，但更改大概率进行不了。
稍后我会修改 Issue 模板，将处理人交给author选择

**Comment by MisakaMikoto521** [comment]
满足第三条，日常积极使用shokax中，可以及时的提出问题和改进，并且及时的进行反馈。然后在新版推出后也能积极进行相应的测试，联系方式的话，在博客主页上有相应的联系方式。
@zkz098

**Reply by zkz098** [reply]
已发送 invite，通过即可加入

**Comment by luhuadong** [comment]
shoka 的老用户，很高兴看到 shokaX 项目，满足第二条，希望能加入一起维护！ @zkz098

**Reply by zkz098** [reply]
已发送 invite，通过即可加入

**Comment by Zjsxstar** [comment]
您好，我是今天使用的这个shoka主题，找不到反馈渠道就在这里说了哈，我现在安装好这个后，主要问题是shoka主题在本地运行时可以看到图片,但是部署到github后，我通过我的博客网址登录看，就显示不出图片了，这个问题不知道咋解决呢，主要是现在您这个主题不支持直接从github上直接下载，themes文件夹下都没有shoka文件，也不知道哪里改= =

**Reply by D-Sketon** [reply]
通过npm下载后shoka(X)主题文件在node_modules中

**Reply by foryouos** [reply]
反馈可以再issue里面进行反馈的，部署到Github不显示，你看一下 您的图片使用的图床，还是，Github本地(直接Github本地的话，国内我之前试的确实不好显示，如果用的图床是没有问题的，如果用的其它例如微信公众号之类的，您需要找到node_modules里面有个head的文件添加反跳转可以解决。

**Reply by Zjsxstar** [reply]
谢谢您的解答，我更正一下哈，那个图片是主题自带的那种，不是我的md文件中的图片哈，我觉得它的图片切换很nice

**Reply by Zjsxstar** [reply]
好滴哈哈哈，但是我没找到shoka这个文件夹
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/51077595/1bdf23d6-8505-435f-a4db-c86c74324a40)

**Reply by D-Sketon** [reply]
看看node_modules里有没有hexo-theme-shokax这个文件夹

**Reply by Zjsxstar** [reply]
找到了，蟹蟹大佬~~

**Reply by Zjsxstar** [reply]
这个问题找到了，说是主题图片是用的其他平台的，建议自己弄个图床，蟹蟹大佬

**Comment by zkz098** [comment]
自今日起第三条需与第一条或第二条结合使用，目前社区维护者人数已经足以支持正常运作

**Comment by Yptrsd** [comment]
满足第二条，详见 9dac86aa2bab318e680e57b97f68636aae9b4af7
（shokaX-docs）
希望帮 ShokaX 的文档变的更加可读。
Thanks！ @zkz098

**Reply by zkz098** [reply]
已发送 invite，通过即可加入
